.eslintrc.json

```diff
@@ -0,0 +1,208 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "ecmaFeatures": {
+      "jsx": true
+    },
```

Comment:
does arborist use jsx? if not, you won't want to allow jsx to parse successfully.